### DISCUSSION TITLE
Donde se entregaran las Katas

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#117] [Q&A] [answered]
Buena tarde compañeros ya casi termino lo primero del front-end donde se subirán las Primeras katas de este modulo

**Comment by BenitoDelaRosaChavez** [comment, marked as answer]
Sera algo parecido a las katas de Python, nos van a dar un link y ahi vamos a poner la información.
